contrib/sandbox_config.json

```diff
@@ -0,0 +1,51 @@
+{
+	"metadata": {
+		"name": "bucketbench",
+		"uid": "bucketbench-test-cri",
+		"namespace": "bucketbench.test.cri",
+		"attempt": 1
+	},
+	"hostname": "crioctl_host",
+	"log_directory": "",
+	"dns_config": {
+		"searches": [
+			"8.8.8.8"
+		]
+	},
+	"port_mappings": [],
+	"resources": {
+		"cpu": {
+			"limits": 3,
+			"requests": 2
+		},
+		"memory": {
+			"limits": 50000000,
+			"requests": 2000000
+		}
+	},
+	"labels": {
+		"group": "test"
+	},
+	"annotations": {
+		"owner": "hmeng",
+		"security.alpha.kubernetes.io/sysctls": "kernel.shm_rmid_forced=1,net.ipv4.ip_local_port_range=1024 65000",
+		"security.alpha.kubernetes.io/unsafe-sysctls": "kernel.msgmax=8192" ,
+		"security.alpha.kubernetes.io/seccomp/pod": "unconfined"
+	},
+	"linux": {
+		"cgroup_parent": "/Burstable/pod_123-456",
```

Comment:
is this `/Burstable/pod_123-456` useful/necessary, or just copied from somewhere else? Maybe it doesn't matter, but would be nice for this to be as "default"/vanilla as can be.

Reply:
I copied it from CRI-O 's contrib files. Will try to make it more generic.